views/env-parts/theme.es

```diff
@@ -1,5 +1,6 @@
 import themes from 'poi-asset-themes/index.json'
 import { remote } from 'electron'
+import { fileUrl } from 'views/utils/tools'
```

Comment:
NewWindow Plugin handles this incorrectly, which resulted in a module not found error.